dumpling/export/dump.go

```diff
@@ -1136,11 +1136,14 @@ func dumpTableMeta(conf *Config, conn *sql.Conn, db string, table *TableInfo) (T
 	return meta, nil
 }
 
-func (d *Dumper) dumpSQL(tctx *tcontext.Context, taskChan chan<- Task) {
+func (d *Dumper) dumpSQL(tctx *tcontext.Context, metaConn *sql.Conn, taskChan chan<- Task) {
 	conf := d.conf
 	meta := &tableMeta{}
 	data := newTableData(conf.SQL, 0, true)
 	task := NewTaskTableData(meta, data, 0, 1)
+	c := detectEstimateRows(tctx, metaConn, fmt.Sprintf("EXPLAIN `%s`", conf.SQL), []string{"rows", "estRows", "count"})
```

Comment:
don't use quotations for sql
```suggestion
	c := detectEstimateRows(tctx, metaConn, fmt.Sprintf("EXPLAIN %s", conf.SQL), []string{"rows", "estRows", "count"})
```